packages/perseus/src/widgets/interactive-graphs/graphs/point.tsx

```diff
@@ -1,17 +1,61 @@
 import * as React from "react";
 
 import {actions} from "../reducer/interactive-graph-action";
+import useGraphConfig from "../reducer/use-graph-config";
 
 import {MovablePoint} from "./components/movable-point";
+import {
+    useTransformDimensionsToPixels,
+    useTransformVectorsToPixels,
+    pixelsToVectors,
+} from "./use-transform";
 
 import type {PointGraphState, MafsGraphProps} from "../types";
 
 type PointGraphProps = MafsGraphProps<PointGraphState>;
 
 export function PointGraph(props: PointGraphProps) {
     const {dispatch} = props;
+    const numPoints = props.graphState.numPoints;
+    const graphState = useGraphConfig();
+    const {
+        range: [[minX, maxX], [minY, maxY]],
+    } = graphState;
+    const width = maxX - minX;
+    const height = maxY - minY;
+    const [[widthPx, heightPx]] = useTransformDimensionsToPixels([
+        width,
+        height,
+    ]);
+    const [[left, top]] = useTransformVectorsToPixels([minX, maxY]);
     return (
         <>
+            {numPoints === "unlimited" && (
```

Comment:
It might be worth having a comment somewhere around here to make it clear what this mysterious `<rect>` is for :)